mirrormanager2/api.py

```diff
@@ -95,3 +95,63 @@ def api_mirroradmins():
     jsonout = flask.jsonify(output)
     jsonout.status_code = 200
     return jsonout
+
+
+@APP.route('/api/repositories/')
+@APP.route('/api/repositories')
```

Comment:
Is something like this always required? Or only in some cases? (just curious, never had to do this)